docs/quickstarts/expo.mdx

```diff
@@ -11,6 +11,10 @@ Learn how to use Clerk to quickly and easily add secure authentication and user
   Currently the `@clerk/expo` SDK only supports mobile app output. You can upvote the addition of Clerk supporting web output [here](https://feedback.clerk.com/roadmap/fb56a8db-763a-4e0d-a4b2-f8a453be195c).
 </Callout>
 
+<Callout type='info' emoji='💡'>
+  The next major version of the Clerk Expo SDK is coming soon! If you're just starting a project, consider [trying out the core 2 beta](https://beta.clerk.com/docs/quickstarts/expo) instead.
```

Comment:
Have we decided not to capitalize "core 2"? I'm fine with it, but thought I'd ask

Reply:
Oh no I think we should capitalize!

Reply:
```suggestion
  The next major version of the Clerk Expo SDK is coming soon! If you're just starting a project, consider [trying out the Core 2 beta](https://beta.clerk.com/docs/quickstarts/expo) instead.
```